lib/ts/recipe/emailpassword/utils.ts

```diff
@@ -81,6 +122,14 @@ export function validateAndNormaliseEmailVerificationConfig(
     config?: TypeInput
 ): TypeNormalisedInputEmailVerification {
     return {
+        emailDelivery:
+            config?.emailDelivery === undefined
+                ? undefined
+                : {
+                      service: config.emailDelivery.service,
+                      // override is undefined here intentionally because the user's override
+                      // for email verification will override the default implementation for emailpassword recipe. So this is not needed.
+                  },
```

Comment:
This is not needed as we will be passing the same emailDelivery instance to emailverification recipe when we create an instance of it in the recipe.ts for emailpassword.